src/xtopcom/xblockmaker/src/xtable_builder.cpp

```diff
@@ -117,7 +117,113 @@ void xlighttable_builder_t::make_light_table_binlog(const xobject_ptr_t<base::xv
 
     make_table_prove_property_hashs(proposal_bstate.get(), property_hashs);
 
-    xdbg("jimmy xlighttable_builder_t::make_light_table_binlog units_size=%zu,sendids=%zu,recvids=%zu,confirmids=%zu,all=%zu,binlog_size=%zu",
+    xdbg("jimmy xlighttable_builder_t::make_light_table_binlog_v1 units_size=%zu,sendids=%zu,recvids=%zu,confirmids=%zu,all=%zu,binlog_size=%zu",
+        units.size(), sendids.size(), recvids.size(), confirmids.size(), all_pairs.size(), property_binlog.size());
+}
+
+void xlighttable_builder_t::make_light_table_binlog_v2(const xobject_ptr_t<base::xvbstate_t> & proposal_bstate,
```

Comment:
do fork in xlighttable_builder_t::make_light_table_binlog better, no change too much